src/galileo/logger/logger.py

```diff
@@ -519,6 +519,59 @@ def has_active_trace(self) -> bool:
         current_parent = self.current_parent()
         return current_parent is not None
 
+    def get_tracing_headers(self) -> dict[str, str]:
+        """
+        Get tracing headers for distributed tracing.
+        Returns X-Galileo-Trace-ID and X-Galileo-Parent-ID headers that can be passed to downstream services.
+
+        Returns
+        -------
+        dict[str, str]
+            Dictionary with "X-Galileo-Trace-ID" and "X-Galileo-Parent-ID" headers.
+            X-Galileo-Parent-ID contains the ID of the current parent (trace or span) that downstream
+            spans should attach to.
+
+        Raises
+        ------
+        GalileoLoggerException
+            If not in streaming mode or if no trace has been started.
+
+        Examples
+        --------
+        ```python
+        logger = GalileoLogger(experimental={"mode": "streaming"})
+        logger.start_trace(input="question")
+        headers = logger.get_tracing_headers()
+        # headers = {"X-Galileo-Trace-ID": "...", "X-Galileo-Parent-ID": "..."}  # trace ID as parent
+
+        logger.add_workflow_span(input="workflow", name="orchestrator")
+        headers = logger.get_tracing_headers()
+        # headers = {"X-Galileo-Trace-ID": "...", "X-Galileo-Parent-ID": "..."}  # workflow span ID as parent
+
+        # Pass headers to HTTP request
+        response = httpx.post(url, headers=headers)
```

Comment:
Which url is this? Is this for users to pass the headers to their own api calls to other services?